code/modules/uplink/uplink_items.dm

```diff
@@ -216,7 +216,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A fully-loaded Aussec Armoury belt-fed machine gun. \
 			This deadly weapon has a massive 50-round magazine of devastating 1.95x129mm ammunition."
 	item = /obj/item/gun/ballistic/automatic/l6_saw
-	cost = 18
+	cost = 19
```

Comment:
???

Reply:
A large gun that holds tons of ammo and can rip apart groups of people with easy